harness/tests/storage/test_storage_base.py

```diff
@@ -14,13 +14,13 @@ def test_unknown_type() -> None:
 
 
 def test_missing_type() -> None:
-    with pytest.raises(CheckFailedError, match="Missing 'type' parameter"):
+    with pytest.raises(check.CheckFailedError, match="Missing 'type' parameter"):
```

Comment:
Would you mind going and changing these checks to just `ValueError` failures instead of the `check` module?

You are already modifying the code right next to the `storage.build()` method, so I think this is an ok time for it.

Reply:
Oh yeah, now I remember that `check` should go away.